docs/conf.py

```diff
@@ -50,6 +49,7 @@
 copyright = "2020, Jennifer Bödker"
```

Comment:
Shouldn'T the copyright lie with QBiC and not the project creator?

Reply:
no, the creator(s) is the copyright holder of the intellectual property.